genon/preprocessor/facade/legacy/BOK_첨부용.py

```
@@ -0,0 +1,1866 @@
+        return tmp_files
+
+    def transcribe_audio(self, file_path_lst: list):
+        transcribed_text_chunks = []
+
+        def _send_request(filepath: str):
+            """Send a request to 'whisper' model served"""
+            files = {
+                'file': (filepath, open(filepath, 'rb'), 'audio/mp3'),
+            }
+
+            response = requests.post(self.req_url, data=self.req_data, files=files)
```

Ruff (0.15.18)

[error] Probable use of `requests` call without timeout

(S113)